### PR TITLE
CMake: OpenMP for Fortran

### PR DESCRIPTION
**Pull request by ax3l**
## Summary

Fix missing activation.

## Additional background

Isolated from #1560

## Checklist

The proposed changes:
- [ ] fix a bug or incorrect behavior in AMReX
- [ ] add new capabilities to AMReX
- [ ] changes answers in the test suite to more than roundoff level
- [ ] are likely to significantly affect the results of downstream AMReX users
- [ ] are described in the proposed changes to the AMReX documentation, if appropriate

### CODE REVIEW

File: Tools/CMake/AMReXParallelBackends.cmake [hunk 1]
```diff
@@ -31,7 +31,7 @@ endif ()
 #
 if (AMReX_OMP)
    set(_omp_comps CXX)
-   if (AMReX_FORTRAN_INTERFACES)
+   if (AMReX_FORTRAN_INTERFACES OR AMReX_FORTRAN)
```
Comment (mic84) [inline Tools/CMake/AMReXParallelBackends.cmake]:
```suggestion
   if (AMReX_FORTRAN)
```
Testing for ``AMReX_FORTRAN_INTERFACES`` is redundant if we also test for ``AMReX_FORTRAN``

**Comment by mic84** [comment]
Can you also update ``AMReXConfig.cmake.in`` accordingly? Thanks!

**Comment by ax3l** [comment]
Good catch - and I just see `set( _mpi_components C CXX )` needs the same patch.

**Comment by mic84** [comment]
I am not sure it does. U

**Comment by ax3l** [comment]
@mic84 did you accidentally close this instead of merging it?

**Review by mic84 — APPROVED** [review]

**Pull request merged**